### DISCUSSION TITLE
Enable `--all-features` in Rust

### DISCUSSION THREAD

**Discussion by otavio** [emacs-lsp/lsp-mode#3892] [Q&A]
Hello, 

I am trying to enable `--all-features` in Rust, but even after adding it to cargo-watch args it still doesn't enable them ... any idea what I am missing?

**Comment by jcs090218** [comment]
May be you should configure `lsp-rust-analyzer-server-command`?

**Reply by brotzeit** [reply]
I think you are looking for `lsp-rust-features`. It's confusing because it's used for rust-analyzer and rls. I remember that we already had this issue a while ago but nobody fixed it.

**Reply by brotzeit** [reply]
Hmmm ok I just realized there's `:checkOnSave` and `:cargo` that contain `:features`. Can you try https://github.com/emacs-lsp/lsp-mode/pull/3893. I think it makes sense to use the same value for both options. At least it reduces potential confusion...

**Reply by brotzeit** [reply]
I updated the pr again. The name of the option has changed.
